docker-compose.yaml

```diff
@@ -13,8 +13,8 @@ services:
       - "8080:8080" # firestore emulator
       - "9199:9199" # storage emulator
       - "4000:4000" # emulator UI
-    working_dir: /home/node/admin
-    command: /bin/bash -c "npm install && npm run emulators:start"
+    working_dir: /home/node/
```

Comment:
Given the makefile: shall we still have the different docker-compose services?
Or should we just have one, with all ports exposed, and then override the run command in the make files?

Reply:
I guess it makes sense to still keep the different Docker compose services. I only see Makefiles as a way to make it easier to work with Docker, but on an application architecture level, it makes sense to me that they are different services in the compose file considering they are also different services on the end-server.